app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessageActivity.kt

```diff
@@ -101,6 +110,45 @@ class MessageActivity : AppCompatActivity() {
             }
         }
 
+        save.setOnClickListener {
+            try {
+                val inputManager: InputMethodManager = this.getSystemService(Context.INPUT_METHOD_SERVICE)
+                        as InputMethodManager
+                inputManager.hideSoftInputFromWindow(content.windowToken, InputMethodManager.SHOW_FORCED)
+
+                val sdf = SimpleDateFormat("dd-MMM-yyyy-hh_mm_aaa")
+                val fileName = "badge-magic-" + sdf.format(Calendar.getInstance().time).toString()
```

Comment:
Will this file be sharable? It is stored inside the app data I guess

Reply:
Yeah for exporting I'll be sending another PR.

Reply:
Great

Reply:
Why not in this same pr, fully complete the feature as you know what is lacking

Reply:
I have already written first steps in the PR title.

Reply:
> I have already written first steps in the PR title.

Oh Sorry, I did not read it. But still you can do it in a single PR